task2.js

```diff
@@ -0,0 +1,19 @@
+const { pipeline } = require("stream");
+const fs = require("fs");
+const csv = require("csvtojson");
+
+const task2 = () => {
+  const inputFile = fs.createReadStream("./csv/nodejs-hw1-ex1.csv");
+  const outputFile = fs.createWriteStream("./csvparsed.txt");
+
+  pipeline(inputFile, csv(), outputFile, (err) => {
```

Comment:
Consider extracting pipeline's callback into a separate function. Such callbacks could be (and usually are) reused between different streams.